Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1212,6 +1202,16 @@ void AXObjectCache::remove(RenderObject& renderer)
 {
     AXTRACE(makeString("AXObjectCache::remove RenderObject* 0x"_s, hex(reinterpret_cast<uintptr_t>(this))));
     remove(m_renderObjectMapping.takeOptional(renderer));
+
+    if (RefPtr node = renderer.node()) {
```

Comment:
If it has a node, then the remove(m_renderObjectMapping.takeOptional(renderer)) above should be a no-op, right? Could we have an assertion that if it has a node, that it's not in m_renderObjectMapping?